lemur/plugins/lemur_acme/plugin.py

```diff
@@ -311,19 +316,32 @@ def get_authorizations(self, acme_client, order, order_info):
         authorizations = []
 
         for domain in order_info.domains:
-            if not self.dns_providers_for_domain.get(domain):
+
+            # If CNAME exists, set host to the target address
+            host = domain
+            if current_app.config.get("ACME_ENABLE_DELEGATED_CNAME", False):
+                val_domain, _ = self.strip_wildcard(domain)
+                val_domain = challenges.DNS01().validation_domain_name(val_domain)
```

Comment:
Maybe this would help with better readability: 
```val_domain --> acme_challenge_val_domain```

```
acme_challenge_val_domain = challenges.DNS01().validation_domain_name(val_domain)
cname_res = self.get_cname(acme_challenge_val_domain)

```

Reply:
This is essentially a temp variable, so I am not sure how much value there is in making it extra descriptive. I went ahead and just made it cname_result the whole way through this code block.

Reply:
thanks. it helps readability.